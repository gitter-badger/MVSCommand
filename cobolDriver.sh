(export STEPLIB=IGY.V5R2M0:$STEPLIB; mvscmd -v --pgm="IGYCRCTL" --args="LIST" --syslin=ibmuser.test.obj\(main\) --sysin=ibmuser.test.c\(main\) --sysprint=*)
