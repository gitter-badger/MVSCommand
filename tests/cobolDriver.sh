tso alloc dsn\("TEST.COBOL.SYSUT1"\) dsorg\(ps\) catalog tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT2"\) dsorg\(ps\) catalog tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT3"\) dsorg\(ps\) catalog tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT4"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT5"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT6"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT7"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT8"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT9"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT10"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT11"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT12"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT13"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT14"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSUT15"\) dsorg\(ps\) catalog  tracks space\(100,10\)
tso alloc dsn\("TEST.COBOL.SYSMDECK"\) dsorg\(ps\) catalog  tracks space\(100,10\)

(export STEPLIB=IGY.V5R2M0:CEE.SCEERUN:CEE.SCEERUN2:$STEPLIB; mvscmd -v --pgm="IGYCRCTL" --args="NONAME,NOTERM,LIST" --syslin=ibmuser.test.obj\(cobol\) --sysin=ibmuser.test.cobol\(test\) --sysprint=ibmuser.test.cobol.list --sysmdeck=ibmuser.test.cobol.sysmdeck \
--sysut1=ibmuser.test.cobol.sysut1 \
--sysut2=ibmuser.test.cobol.sysut2 \
--sysut3=ibmuser.test.cobol.sysut3 \
--sysut4=ibmuser.test.cobol.sysut4 \
--sysut5=ibmuser.test.cobol.sysut5 \
--sysut6=ibmuser.test.cobol.sysut6 \
--sysut7=ibmuser.test.cobol.sysut7 \
--sysut8=ibmuser.test.cobol.sysut8 \
--sysut9=ibmuser.test.cobol.sysut9 \
--sysut10=ibmuser.test.cobol.sysut10 \
--sysut11=ibmuser.test.cobol.sysut11 \
--sysut12=ibmuser.test.cobol.sysut12 \
--sysut13=ibmuser.test.cobol.sysut13 \
--sysut14=ibmuser.test.cobol.sysut14 \
--sysut15=ibmuser.test.cobol.sysut15\
)

tsocmd delete "TEST.COBOL.SYSUT1"
tsocmd delete "TEST.COBOL.SYSUT2"
tsocmd delete "TEST.COBOL.SYSUT3"
tsocmd delete "TEST.COBOL.SYSUT4"
tsocmd delete "TEST.COBOL.SYSUT5"
tsocmd delete "TEST.COBOL.SYSUT6"
tsocmd delete "TEST.COBOL.SYSUT7"
tsocmd delete "TEST.COBOL.SYSUT8"
tsocmd delete "TEST.COBOL.SYSUT9"
tsocmd delete "TEST.COBOL.SYSUT10"
tsocmd delete "TEST.COBOL.SYSUT11"
tsocmd delete "TEST.COBOL.SYSUT12"
tsocmd delete "TEST.COBOL.SYSUT13"
tsocmd delete "TEST.COBOL.SYSUT14"
tsocmd delete "TEST.COBOL.SYSUT15"
tsocmd delete "TEST.COBOL.SYSMDECK"
