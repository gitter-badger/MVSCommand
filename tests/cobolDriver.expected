alloc dsn(TEST.COBOL.SYSUT1) dsorg(ps) catalog tracks space(100,10)
RC(12)
alloc dsn(TEST.COBOL.SYSUT2) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT3) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT4) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT5) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT6) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT7) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT8) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT9) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT10) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT11) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT12) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT13) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT14) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSUT15) dsorg(ps) catalog tracks space(100,10)
alloc dsn(TEST.COBOL.SYSMDECK) dsorg(ps) catalog tracks space(100,10)
Program: <IGYCRCTL> Arguments: <NONAME,NOTERM,LIST>
DDNames:
  SYSUT15=IBMUSER.TEST.COBOL.SYSUT15
  SYSUT14=IBMUSER.TEST.COBOL.SYSUT14
  SYSUT13=IBMUSER.TEST.COBOL.SYSUT13
  SYSUT12=IBMUSER.TEST.COBOL.SYSUT12
  SYSUT11=IBMUSER.TEST.COBOL.SYSUT11
  SYSUT10=IBMUSER.TEST.COBOL.SYSUT10
  SYSUT9=IBMUSER.TEST.COBOL.SYSUT9
  SYSUT8=IBMUSER.TEST.COBOL.SYSUT8
  SYSUT7=IBMUSER.TEST.COBOL.SYSUT7
  SYSUT6=IBMUSER.TEST.COBOL.SYSUT6
  SYSUT5=IBMUSER.TEST.COBOL.SYSUT5
  SYSUT4=IBMUSER.TEST.COBOL.SYSUT4
  SYSUT3=IBMUSER.TEST.COBOL.SYSUT3
  SYSUT2=IBMUSER.TEST.COBOL.SYSUT2
  SYSUT1=IBMUSER.TEST.COBOL.SYSUT1
  SYSMDECK=IBMUSER.TEST.COBOL.SYSMDECK
  SYSPRINT=IBMUSER.TEST.COBOL.LIST
  SYSIN=IBMUSER.TEST.COBOL(TEST)
  SYSLIN=IBMUSER.TEST.OBJ(COBOL)
Dynalloc succeeded for SYSUT15=IBMUSER.TEST.COBOL.SYSUT15
Dynalloc succeeded for SYSUT14=IBMUSER.TEST.COBOL.SYSUT14
Dynalloc succeeded for SYSUT13=IBMUSER.TEST.COBOL.SYSUT13
Dynalloc succeeded for SYSUT12=IBMUSER.TEST.COBOL.SYSUT12
Dynalloc succeeded for SYSUT11=IBMUSER.TEST.COBOL.SYSUT11
Dynalloc succeeded for SYSUT10=IBMUSER.TEST.COBOL.SYSUT10
Dynalloc succeeded for SYSUT9=IBMUSER.TEST.COBOL.SYSUT9
Dynalloc succeeded for SYSUT8=IBMUSER.TEST.COBOL.SYSUT8
Dynalloc succeeded for SYSUT7=IBMUSER.TEST.COBOL.SYSUT7
Dynalloc succeeded for SYSUT6=IBMUSER.TEST.COBOL.SYSUT6
Dynalloc succeeded for SYSUT5=IBMUSER.TEST.COBOL.SYSUT5
Dynalloc succeeded for SYSUT4=IBMUSER.TEST.COBOL.SYSUT4
Dynalloc succeeded for SYSUT3=IBMUSER.TEST.COBOL.SYSUT3
Dynalloc succeeded for SYSUT2=IBMUSER.TEST.COBOL.SYSUT2
Dynalloc succeeded for SYSUT1=IBMUSER.TEST.COBOL.SYSUT1
Dynalloc succeeded for SYSMDECK=IBMUSER.TEST.COBOL.SYSMDECK
Dynalloc succeeded for SYSPRINT=IBMUSER.TEST.COBOL.LIST
Dynalloc succeeded for SYSIN=IBMUSER.TEST.COBOL(TEST)
Dynalloc succeeded for SYSLIN=IBMUSER.TEST.OBJ(COBOL)
OS Load program IGYCRCTL
Addressing mode: AMODE31
Exit code: 0 from IGYCRCTL
delete TEST.COBOL.SYSUT1
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT1 DELETED
delete TEST.COBOL.SYSUT2
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT2 DELETED
delete TEST.COBOL.SYSUT3
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT3 DELETED
delete TEST.COBOL.SYSUT4
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT4 DELETED
delete TEST.COBOL.SYSUT5
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT5 DELETED
delete TEST.COBOL.SYSUT6
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT6 DELETED
delete TEST.COBOL.SYSUT7
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT7 DELETED
delete TEST.COBOL.SYSUT8
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT8 DELETED
delete TEST.COBOL.SYSUT9
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT9 DELETED
delete TEST.COBOL.SYSUT10
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT10 DELETED
delete TEST.COBOL.SYSUT11
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT11 DELETED
delete TEST.COBOL.SYSUT12
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT12 DELETED
delete TEST.COBOL.SYSUT13
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT13 DELETED
delete TEST.COBOL.SYSUT14
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT14 DELETED
delete TEST.COBOL.SYSUT15
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSUT15 DELETED
delete TEST.COBOL.SYSMDECK
IDC0550I ENTRY (A) IBMUSER.TEST.COBOL.SYSMDECK DELETED
